tests/test_multi_mapping.py

```diff
@@ -0,0 +1,261 @@
+"""
+these tests check the multi-mapping needs for integration tests.
+"""
+
+import pytest
+
+from pathlib import Path
+
+
+import clicktools
+import csvrow
+
+print("should this test people with birthdate on separte line?")
+
+
+@pytest.mark.unit
+def test_mapping_person(tmp_path: Path):
+    arg__input_dir = Path(__file__).parent / "test_multi_mapping/test_mapping_person"

```

Comment:
Worth adding even brief one line docstrings to describe why these tests exists for future maintainers